share/scan/apps.go

```diff
@@ -781,6 +787,16 @@ func (s *ScanApps) parseDotNetPackage(filename, fullpath string) {
 	}
 }
 
+func getNetModuleName(input string) string {
```

Comment:
I believe renaming it to `getDotNetModule` would enhance readability.
Would it be possible for you to add a unit test to cover this as well? That would be greatly appreciated!